src/Acuminator/Acuminator.Utilities/Roslyn/Semantic/PXGraph/Infos/DataViewInfo.cs

```diff
@@ -20,6 +20,8 @@ public class DataViewInfo : GraphNodeSymbolItem<ISymbol>
 		/// The type of the data view symbol
 		/// </summary>
 		public INamedTypeSymbol Type { get; }
+		
+		public ITypeSymbol ViewDAC { get; }
```

Comment:
What is the purpose of this property?

Reply:
It stores the DAC of the view. For example:
```
PXSelect<TDAC> -view
TDAC - ViewDAC
```
 It's quite convenient to have this info in DTO instead of retrieving it from context